CONTRIBUTING.md

```diff
@@ -49,6 +49,17 @@ To make edits to the HTML generation, run the following commands:
     opam install --deps-only odoc
     ```
 
+    Also make sure to install a recent version of `tidy-html5` (required for
```

Comment:
Could we add a link to the tidy website? (http://www.html-tidy.org/)

Reply:
Also note that tidy is not required for the test suite. It's optional, the test suite detects it and uses it if installed.

It is used for HTML validity testing, not for pretty-printing.

Reply:
@aantron So it seems that for MacOS, `tidy` is installed by default, which caused my tests to fail because of a missing CLI flag. Don't you think it should be installed, no matter what?

Reply:
We can certainly put that in the instructions. Thanks for doing it :)